cni/cni.go

```diff
@@ -14,20 +14,21 @@ import (
 	network "github.com/Microsoft/windows-container-networking/network"
 	cniSkel "github.com/containernetworking/cni/pkg/skel"
 	cniTypes "github.com/containernetworking/cni/pkg/types"
-	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
+	cniTypesCurr "github.com/containernetworking/cni/pkg/types/100"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	// CNI commands.
-	CmdAdd = "ADD"
-	CmdDel = "DEL"
+	CmdAdd   = "ADD"
+	CmdDel   = "DEL"
+	CmdCheck = "CHECK"
 
 	Internal = "internal"
 )
 
 // Supported CNI versions.
-var VersionsSupported = []string{"0.2.0", "0.3.0"}
+var VersionsSupported = []string{"0.2.0", "0.3.0", "1.0.0"}
```

Comment:
We can also add 0.4.0 here? Any additional changes required for 0.4.0?

Reply:
Good catch! I have just added `0.4.0` to the list as well!

> Any additional changes required for 0.4.0?

As per [update guidelines for 0.4.0](https://github.com/containernetworking/cni/blob/main/Documentation/spec-upgrades.md#changes-in-cni-v04), all that is needed from the plugin end was handling `CHECK`. (I had initially forgotten to [hook it into the CniSkel](https://github.com/microsoft/windows-container-networking/pull/101/files#diff-62698665e85b9e578006a98b0335b734e0ea9bf88c2704cb237c0fc76014a61eR48) but have fixed that)

The [update guidelines for 1.0.0](https://github.com/containernetworking/cni/blob/main/Documentation/spec-upgrades.md#libcni-changes-in-cni-v10) only mentions [switching away from `types/current` to `types/XYZ`](https://github.com/microsoft/windows-container-networking/pull/101/files#diff-8a47859b51210f40efd508827dea21a220fc7c0d7fee3f8d2a87ec38220a26baL17).